src/GitVersionExe.Tests/GitVersionHelper.cs

```diff
@@ -2,26 +2,33 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Text;
+
```

Comment:
Is this r# splitting it up? We should add the setting to stop this automatically happening (for consistency reasons) into the solution DotSettings file